Kubernetes/OrderForm/dev-orderform-kubernetes-manifest.yml

```diff
@@ -54,39 +58,54 @@ spec:
 
           ## The base uri of identity service
         - name: OIDC_BASE_URI
-          value: srt-oidcbaseuri@azurekeyvault
+          value: "https://buyingcataloguedev.uksouth.cloudapp.azure.com/identity"
 
           ## The client id to be sent to identity service
         - name: OIDC_CLIENT_ID
-          value: srt-oidcclientid@azurekeyvault
+          valueFrom:
+            secretKeyRef:
+              name: devpipelinevariables
+              key: OIDC_CLIENT_ID
 
           ## The secret need to decode JWT tokens
         - name: OIDC_CLIENT_SECRET
-          value: srt-oidcclientsecret@azurekeyvault
+          valueFrom:
+            secretKeyRef:
+              name: devpipelinevariables
+              key: OIDC_CLIENT_SECRET
+
+          ##How long before the cookies stored in the session expire in ms
+        - name: MAX_COOKIE_AGE
+          value: "3600000"
+
+          ##The secret needed for encoding and decoding the cookie
+        - name: COOKIE_SECRET
+          valueFrom:
+            secretKeyRef:
+              name: devpipelinevariables
+              key: COOKIE_SECRET
 
           ## The Host in which redis is running
         - name: REDIS_URL
-          value: srt-redishost@azurekeyvault
+          valueFrom:
+            secretKeyRef:
+              name: devpipelinevariables
+              key: REDIS_HOST
 
           ## The port redis is running
         - name: REDIS_PORT
-          value: srt-redisport@azurekeyvault
+          value: "6380"
 
           ## Boolean to indicate whether to connect to redis via TLS
         - name: REDIS_TLS
-          value: srt-redistls@azurekeyvault
+          value: "true"
 
           ## The password to connect to redis
         - name: REDIS_PASS
-          value: srt-redispass@azurekeyvault
-
-          ## The secret needed for encoding and decoding the cookie
-        - name: COOKIE_SECRET
```

Comment:
does order form still need the cookie secret?

Reply:
I would assume so as OF needs to log users in, why?

Reply:
I can see that it's not been removed, it's just moved up the file. Thanks for pointing that out @assoomro